src/utils/configuration.ts

```diff
@@ -48,9 +48,10 @@ function parseArguments(args: any): { [key: string]: string } {
                     return [...acc, { key: curr.key.name, value: curr.value.value }];
                 }, []);
             } else {
-                throw new Error("Unsupported argument for decorator");
+                return undefined;
             }
         })
+        .filter((a: any) => a !== undefined)
```

Comment:
Please use unknown instead of any.
```suggestion
        .filter((a: unknown) => a !== undefined)
```